integration/bin/tachyon-yarn-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# prepare Tachyon dir structure for a YARN container to launch master or worker
+tar zxf tachyon.tar.gz
+
+# launch master or worker on this container
+if [ $1 = 'master' ]; then
+  ./integration/bin/tachyon-master-yarn.sh
```

Comment:
ah, I guess this script will run before TACHYON_HOME exists ... I guess in that case it is fine to use the relative path